### PR TITLE
Fix Validations for Transfer To Admin

### PR DESCRIPTION
**Pull request by dimak1**
*Issue #:* 
- bcgov/entity#18382

*Description of changes:*
- Fix validation for Transfer to Admin (LETA)

This PR fixed another issue:
- bcgov/entity#19157

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/mhrTransfers/SupportingDocuments.vue [hunk 1]
```diff
@@ -151,6 +153,10 @@ export default defineComponent({
   display: flex;
   flex-direction: column;
 
+  :deep(.non-clickable-label .v-label) {
+    pointer-events: none;
```
Comment (dimak1) [inline ppr-ui/src/components/mhrTransfers/SupportingDocuments.vue:157]: This prevents clicking on the label of the Supporting Document radio buttons. When clicking on label, and not on the radio button itself, was causing issues with wrong selections, which I could not figure out.

**Review by cameron-eyds — APPROVED** [review]
Looks good!

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1794-kdt26ink.web.app

**Review by doug-lovett — APPROVED** [review]

**Pull request merged**